### ISSUE TITLE
[Feature and Bug] Seekbar, Hold Menu, Notification Animation

### ISSUE THREAD

**Issue by seniorm0ment** [NO-ob/LoliSnatcher_Droid#75]
# Bug

The seek bar is open when swiping between videos.

# Bug 2

The new animation that appears at the bottom of the screen covers up the buttons to save, see file info, and more.

# Suggestion 

Suggesting fixing the seek bar being open everytime the user swipes between videos.

I also suggest moving all the buttons from the press to hold menu (save, file info, etc buttons) directly onto the playback bar box. All the icons can fit.

Also suggesting adjusting the notification at the bottom and maybe make it floating so it doesn't interfere with buttons.

**Comment by NANI-SORE** [comment]
I assume you are on the prerelease build @NO-ob sent you, thanks for testing it.
- Seek bar: If you are talking about it being always visible between swipes until you tap, I know about that and it is in my list of things to fix later.
- Notifications covering up stuff: already fixed it before I read this issue, they will be above the toolbar and some of them will be appearing from the top in the final release.
- Moving toolbar into video controls: It's not possible since the video controls are a part of the player component. And in my opinion video controls shouldn't be mixed with toolbar actions, it will just become too cramped in there and everything will go in overflow menu, which will cause unnecessary steps to reach some actions.

**Comment by seniorm0ment** [comment]
Yes and ok, and as far as the toolbar controls go, I don't disagree but I just feel personally it feels out of place and when it's open plus the video controls it takes up a lot of screen space imo.
That and the fact that those open with a long hold, and the player controls is a single tap.
Not sure what could be done, maybe move them above the player controls (or to the top of the screen) while making the background of that area almost transparent? Just idea.
Thanks
